libs/application/template-api-modules/src/lib/modules/templates/social-insurance-administration/social-insurance-administration.service.ts

```diff
@@ -168,6 +187,60 @@ export class SocialInsuranceAdministrationService extends BaseTemplateApiService
     return attachments
   }
 
+  private async getHSAttachments(
```

Comment:
We should definitely provide a way to gather attachments from applications that is reusable across all applications. I will put that on the todo list for us.